src/formatDate.js

```diff
@@ -48,9 +48,28 @@
  *
  * @returns {string}
  */
-
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatArr = date.split(`${fromFormat[3]}`);
```

Comment:
You're using template strings unnecessarily. There's no need to use them if you're not including any variables or expressions. Instead of `${fromFormat[3]}` you can just use fromFormat[3].